teachopencadd/talktorials/T033_molecular_representations/talktorial.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<p>re:</p><blockquote>talktorial <a href="https://github.com/volkamerlab/teachopencadd/tree/master/teachopencadd/talktorials" target="_blank">T034 · SMILES based property prediction</a></blockquote><p> </p><p>Throughout TeachOpenCADD, we always referred to other talktorials without any links (as they are different between website and GH views &amp; in case of new talktorials not yet existent as not on master yet).</p><p>We simply write <strong>Talktorial T005</strong> (bold, capitalised T in Talktorial, and index including 0s, e.g. T005 or T010). See third blue box in template: https://github.com/volkamerlab/teachopencadd/blob/master/teachopencadd/talktorials/T000_template/talktorial.ipynb</p><p> </p><p> </p><p>also in general smaller notebook cells make reviews easier (we broke up longer cells into multiple shorter cells).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>